leader-election/DistributedMutex/BlobLeaseManager.cs

```diff
@@ -105,41 +108,45 @@ public async Task<bool> RenewLeaseAsync(string leaseId, CancellationToken token)
         {
             try
             {
-                await this.leaseBlob.RenewLeaseAsync(new AccessCondition { LeaseId = leaseId }, token);
+                if (this.leaseClient == null)
+                {
+                    this.leaseClient = this.leaseBlobClient.GetBlobLeaseClient(leaseId);
```

Comment:
Consider refactoring so ghte GetBlobLeaseClient call is not repeated in this example.  Low pri and please push back if not necessary.

Reply:
It will be called once per worker role at the first time.